tdrs-backend/tdpservice/settings/common.py

```diff
@@ -114,44 +113,21 @@ class Common(Configuration):
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 
     # Whether to use localstack in place of a live AWS S3 environment
-    USE_LOCALSTACK = bool(os.getenv("USE_LOCALSTACK", 0))
-
-    # AWS Access Keys
-    AWS_S3_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
-    AWS_S3_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-    AWS_STORAGE_BUCKET_NAME = os.getenv("AWS_BUCKET")
-    AWS_REGION_NAME = os.getenv("AWS_REGION_NAME")
+    USE_LOCALSTACK = bool(strtobool(os.getenv("USE_LOCALSTACK", "no")))
```

Comment:
Changed this to use a string and the intermediary function `strtobool` to prevent an error where the string `0` would be counted as True and Localstack would be used anyway even though it was turned off.